subsys/bluetooth/mesh/subnet.h

```diff
@@ -36,6 +36,7 @@ struct bt_mesh_net_cred {
 /** Subnet instance. */
 struct bt_mesh_subnet {
 	uint32_t beacon_sent;        /* Timestamp of last sent beacon */
+	uint32_t beacon_recv;		 /* Timestamp of last received beacon */
```

Comment:
The alignment of the code comment is wrong: all other surrounding lines use spaces, but you used tabs (those should only ever exist at the very beginning of a line, and spaces should be used elsewhere).